akka-persistence-tck/src/main/scala/akka/persistence/journal/JournalPerfSpec.scala

```diff
@@ -61,6 +71,40 @@ object JournalPerfSpec {
 
   case object ResetCounter
   case class Cmd(mode: String, payload: Int)
+
+  /**
+   * INTERNAL API
+   */
+  @InternalApi private[akka] class CmdSerializer extends SerializerWithStringManifest {
+    override def identifier: Int = 293562
+
+    override def manifest(o: AnyRef): String = ""
+
+    override def toBinary(o: AnyRef): Array[Byte] =
+      o match {
+        case Cmd(mode, payload) =>
+          s"$mode|$payload".getBytes(StandardCharsets.UTF_8)
```

Comment:
so `mode` is no longer allowed to contain a `|`. Might be fine since it's a TCP feature anyway.

Reply:
yes, it's the TCK that creates the `Cmd`s so shouldn't be a problem